apps/judicial-system/backend/migrations/20210311090927-update-case.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) =>
+      queryInterface.addColumn(
+        'case',
+        'isolation_to',
+        {
+          type: 'TIMESTAMP WITH TIME ZONE',
+          defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
```

Comment:
```suggestion
          type: Sequelize.DATE,
```

We use timestamps for 'created' and 'modified' but simple 'date' for other date fields and no default values.